test/test-network/systemd-networkd-tests.py

```diff
@@ -7805,5 +7805,11 @@ def test_mtu_link_ipv6_mtu(self):
     if enable_debug:
         wait_online_env.update({'SYSTEMD_LOG_LEVEL': 'debug'})
 
-    sys.argv[1:] = unknown_args
-    unittest.main(verbosity=3)
+    unittest.main(
+        verbosity=3,
+        argv=[
+            sys.argv[0],
+            *unknown_args,
+            *(["-k", match] if (match := os.getenv("TEST_MATCH_TESTCASE")) else [])
```

Comment:
Not bad, but `-k` is overkill, I think. Why not simply pass TEST_MATCH_TESTCASE?

Reply:
Because test-functions matches on regex as well so I wanted to do the closest thing for the networkd tests.